.github/workflows/ci-cd-release-please.yml

```
@@ -6,7 +6,7 @@
 # 2.3. Triggers staging and yt01 deployment via repository dispatch
 
 name: CI/CD Release Please
-
+description: When new releases are created, this workflow builds and publishes docker images and notify other environments about it through dispatch
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)